R/get-submission-metadata.R

```diff
@@ -67,6 +67,8 @@ get_submissions_metadata <- function(syn, group, all_users = TRUE,
 #'
 #' @keywords internal
 #' @param data The json submission metadata.
+#' @importFrom magrittr %>%
```

Comment:
Function-level lines like this was what I was referring to. Are these auto-generated by roxygen? If not, I think `importFrom(magrittr,"%>%")` in the NAMESPACE and specifying magrittr in the imports is sufficient.

Reply:
The comments are not auto generated by roxygen, but they are what roxygen uses to generate the NAMESPACE. So they need to be here if we want it added to the NAMESPACE.

Reply:
Ah, interesting. So just a single inclusion (in the function docstring) is enough for Roxygen to put pipe in the package namespace, and thus make it available to all functions? Quirky 🤷

Reply:
yup